src/System.Runtime.WindowsRuntime/src/Resources/Strings.resx

```diff
@@ -348,4 +348,7 @@
   <data name="UnauthorizedAccess_InternalBuffer" xml:space="preserve">
     <value>MemoryStream's internal buffer cannot be accessed.</value>
   </data>
+  <data name="Argument_NeedNonNegNum" xml:space="preserve">
+    <value>Non-negative number required.</value>
```

Comment:
Resource string removed, using message to go with switch to `ArgumentOutOfRange`